static/css/restyle/restyle.less

```diff
@@ -1655,18 +1669,45 @@ h3.author .transfer-ownership {
   text-shadow: none;
 }
 
+.addon.hovercard,
 .items .item.incompatible {
+
```

Comment:
Eventually we'll need an issue to convert all selectors to have this newline or not ^_^